### PR TITLE
Docker & environment code. 

### PR DESCRIPTION
**Pull request by mckadesorensen**
The docker file will prevent issues that arise from having different OS and OS versions.

### CODE REVIEW

File: Build/buen_aire_backend.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+RUN yum install -y gcc git make awscli
+
+RUN yum install -y python3-devel
+
+ADD requirements.txt /requirements/
+
+# Python 3 code dependencies
+RUN pip3 install -r /requirements/requirements.txt
+
+WORKDIR /Buen-Aire-Backend
```
Comment (mckadesorensen) [inline Build/buen_aire_backend.Dockerfile:13]: Add a new line at the end of the file homie

Comment (ETY-13) [reply Build/buen_aire_backend.Dockerfile:13]: OK.

**Review by ETY-13 — COMMENTED** [review]
looks good.

File: Build/buen_aire_backend.Dockerfile (hunk 1 above)
Comment (mckadesorensen) [reply Build/buen_aire_backend.Dockerfile:13]: I think I just said that to demo something to you guys when we talked about git, lol.

**Review by colonelkernell — APPROVED** [review]

**Pull request merged**